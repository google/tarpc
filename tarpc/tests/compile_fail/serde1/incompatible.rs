#![allow(deprecated)]
#[tarpc::service(derive = [Clone], derive_serde = true)]
trait Foo {
    async fn foo();
}

fn main() {}
