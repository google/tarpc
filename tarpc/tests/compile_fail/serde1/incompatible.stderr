error: tarpc does not support `derive_serde` and `derive` at the same time
 --> tests/compile_fail/serde1/incompatible.rs:2:1
  |
2 | #[tarpc::service(derive = [Clone], derive_serde = true)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `tarpc::service` (in Nightly builds, run with -Z macro-backtrace for more info)
